osa/parser.js

```diff
@@ -10,6 +10,13 @@ const Parser = require('../.lib/parser.js');
  * @param  {Object} ec        an object representing the EC whose URL is being analyzed
  * @return {Object} the result
  */
+
+function parseURIParam(uri, result) {
```

Comment:
I updated this function to return the attributes instead of assigning them directly to a parameter